processor/src/main/java/org/mapstruct/ap/internal/model/ContainerMappingMethod.java

```diff
@@ -36,11 +36,8 @@
  *
  * @author Filip Hrisafov
  */
-public abstract class ContainerMappingMethod extends MappingMethod {
```

Comment:
You mean `loopVariableName`, right? This one is actually needed:
* In `IterableMappingMethod.ftl` - here it is clear
* In `StreamMappingMethod.ftl` -for updating an array on line 103

Both of them have some form of element assignment and use the `SelectionParameters`. Theoretically other similar mappings (for example Guava collections) can benefit from the `ContainerMappingMethod`.

As for `getDefault()`, it was in the `IterableMappingMethod` and I pulled it up when I needed it for the `StreamMappingMethod`. I didn't know about `Type#getNull()`, had I known I would have used it. I can clean that up as well and use the one from the `Type`. We will just have to agree on the differences between the 2.